scripts/release.py

```diff
@@ -433,16 +433,17 @@ def generate_commit_table(start_ref, end_ref):
     The commits in the table should only be commits that are not in the
     pull request table.
     """
-    header = "|| Author || Summary || Commit || JIRA || Verified? ||"
+    header = "|| Author || Summary || Commit || JIRA || RN? || Verified? ||"
```

Comment:
I'm a bit concerned that it will take up too much horizontal space, but it might be worth it to reduce confusion. I've made the change.